scale/product/models.py

```diff
@@ -242,6 +277,16 @@ def filter_products(self, started=None, ended=None, job_type_ids=None, job_type_
             products = products.filter(is_superseded=is_superseded)
         if file_name:
             products = products.filter(file_name=file_name)
+        if job_output:
+            products = products.filter(job_output=job_output)
+        if recipe_ids:
+            products = products.filter(recipe_id__in=recipe_ids)
+        if recipe_job:
+            products = products.filter(recipe_job=recipe_job)
+        if recipe_type:
+            products = products.filter(recipe_type=recipe_type)
```

Comment:
Shouldn't *recipe_type* be *recipe_type_ids* and an integer list instead of a string? That would match the REST API docs and how *job_type_ids* was done.